tests/python_tests/helpers/profiler.py

```diff
@@ -51,18 +51,22 @@ def _process_profiler_message(line: str):
     )
 
 
+def _assert_no_collision(messages, message):
+    """Inserts message if no collisions, raises otherwise"""
+    collision = messages.get(message.id)
+    if collision and collision != message:
+        raise AssertionError(f'Hash collision between "{message}" and "{collision}"')
```

Comment:
```suggestion
    existing = messages.get(message.id)
    if existing is not None and existing != message:
        raise AssertionError(f'Hash collision between "{message}" and "{existing}"')
```
More Pythonic way of writing it, nothing wrong with the initial version.